src/muya/lib/contentState/pasteCtrl.js

```diff
@@ -99,9 +99,15 @@ const pasteCtrl = ContentState => {
       const text = link.textContent
 
       if (href === text) {
-        const span = document.createElement('span')
-        span.innerHTML = text
-        link.replaceWith(span)
+        const title = await getPageTitle(href)
```

Comment:
@Jocs Does this may cause problems when typing other stuff and `getPageTitle` take a long time or times out? Or is the editor block for this time?

E.g.

1. Paste text with link in it
2. `getPageTitle` http request (async)
3. User: type random words
4. http request (not) successful
5. ???

Reply:
Yes, it does not be blocked, but it maybe take some time to request the page title, I set a timeout of 1.5 s, but it shouldn't be perfect. It's better to insert the link first, and then insert the anchor text (title content) after the title is obtained.

I will optimize before the 1.6 release.

Reply:
Yes, it does not be blocked, but it maybe take some time to request the page title, I set a timeout of 1.5 s, but it shouldn't be perfect. It's better to insert the link first, and then insert the anchor text (title content) after the title is obtained.

I will optimize before the 1.6 release.